src/legacy/core_plugins/dashboard_embeddable_container/public/np_ready/public/lib/embeddable/dashboard_container.test.tsx

```diff
@@ -144,62 +138,3 @@ test('Container view mode change propagates to new children', async () => {
 
   expect(embeddable.getInput().viewMode).toBe(ViewMode.EDIT);
 });
-
-test('DashboardContainer in edit mode shows edit mode actions', async () => {
```

Comment:
I did not delete it. Moved it to `src/legacy/core_plugins/dashboard_embeddable_container/public/np_ready/public/tests/dashboard_container.test.tsx` as other tests in this file are like unit tests, but this test case is quite involved.

Reply:
Oh I see this response now.  maybe instead of moving this one test to a separate folder, just rename it something like `dashboard_container_actions.test.tsx`, and leave in this folder?  I think that would make the reason for the separation a little clearer.